Sources/SentryCrash/Recording/SentryCrashReportFixer.c

```diff
@@ -233,7 +233,7 @@ sentrycrashcrf_fixupCrashReport(const char *crashReport)
         .onNullElement = onNullElement,
         .onStringElement = onStringElement,
     };
-    int stringBufferLength = 10000;
+    int stringBufferLength = 100000;
```

Comment:
`m`: if those two variables depend on each other, wouldn't it make sense to create a constant for them and reuse it?